website/docs/Examples/EMGProjects/EMG_Joystick.md

```diff
@@ -0,0 +1,67 @@
+---
+id: EMG_Joystick
+title: EMG Joystick
+---
+This tutorial will show you how to control a joystick using EMG data with the OpenBCI GUI. You can then use this muscle activated joystick for any purpose of your choosing! We have developed an tuneable adaptive EMG classifier for accurate control of the joystick. We used this to control a drone as seen in the 2023 OpenBCI TED Talk.
```

Comment:
- use this customizable, muscle-activated joystick for any
- It's not technically a classifier by definition. Cut "We have developed ... of the joystick" sentence. See previous bullet point that achieves same goal by adding word "customizable"